### PR TITLE
Bunch o fixes

### PR DESCRIPTION
**Pull request by Efimero**
mostly typos and clarifications

### CODE REVIEW

File: docs/Vanilla/Recipes/Crafting/ICraftingRecipe.md [hunk 1]
```diff
@@ -42,11 +42,33 @@ Parameter Types are [ICraftingInventory](ICraftingInventory) and [IPlayer](/Vani
...
+### resourceDomain
+
+Basically, the modid of the mod that added the recipe.
+```
+rec.resourceDomain;
+```
+
+### Ingredients
+
+Returns the ingredients list as [IIngredient](IIngredient)\[] or [IIngredient](IIngredient)\[]\[] respectively.
```
Comment (kindlich) [inline docs/Vanilla/Recipes/Crafting/ICraftingRecipe.md]: Change the link to be `/Vanilla/Variable_Types/IIngredient`

File: docs/Vanilla/Recipes/Crafting/ICraftingRecipe.md [hunk 2]
```diff
@@ -42,11 +42,33 @@ Parameter Types are [ICraftingInventory](ICraftingInventory) and [IPlayer](/Vani
...
+```
+rec.ingredients1D;
+rec.ingredients2D;
+```
+
+### Output
+
+The [IItemStack](IItemStack) output of the recipe.
```
Comment (kindlich) [inline docs/Vanilla/Recipes/Crafting/ICraftingRecipe.md]: Change the link to `/Vanilla/Items/IItemStack`

File: docs/Mods/Artisan_Worktables/CraftTweaker_Support/To_Scale_Example.md [hunk 3]
```diff
@@ -6,44 +6,42 @@ Here's a Quick To Scale Example:
...
-			  <minecraft:dark_oak_stairs>] as IItemstack[];
-
-var StairsRecipe = Worktable.createRecipeBuilder("carpenter");
-
-StairsRecipe.setShaped([
+	      <minecraft:birch_stairs>,
+	      <minecraft:spruce_stairs>,
+	      <minecraft:jungle_stairs>,
+	      <minecraft:acacia_stairs>,
+	      <minecraft:dark_oak_stairs>] as IItemstack[];
```
Comment (kindlich) [inline docs/Mods/Artisan_Worktables/CraftTweaker_Support/To_Scale_Example.md]: I know that's not your doing, but could you add the import to IItemStack to the example as well?

File: docs/Vanilla/Recipes/Crafting/ICraftingRecipe.md [hunk 4]
```diff
@@ -42,11 +42,33 @@ Parameter Types are [ICraftingInventory](ICraftingInventory) and [IPlayer](/Vani
 rec.applyTransformers(ICraftingInventory inventory, IPlayer byPlayer);
 ```
 
+### resourceDomain
+
+Basically, the modid of the mod that added the recipe.
+```
+rec.resourceDomain;
```
Comment (kindlich) [inline docs/Vanilla/Recipes/Crafting/ICraftingRecipe.md]: Could you add the `fullResourceDomain` getter as well?

**Comment by Efimero** [comment]
updated with requested changes

**Pull request merged**